specs/Standards/Tokens/FungibleTokenMetadata.md

```diff
@@ -90,8 +90,8 @@ type FungibleTokenMetadata = {
 
 ## Drawbacks
 
-- Enforcement of `icon` to be a data URL rather than a link to an HTTP endpoint that could contain privacy-violating code cannot be done on deploy or update of contract metadata, and must be done on the consumer/app side when displaying token data.
-- If on-chain icon uses a data URL or is not set but the document given by `reference` contains a privacy-violating `icon` URL, consumers & apps of this data should not naïvely display the `reference` version, but should prefer the safe version. This is technically a violation of the "`reference` setting wins" policy described above.
+- It could be argued that `symbol` and even `name` could belong as key/values in the `reference` JSON object.
+- It might make sense to add `icon_url` and `icon_hash` to the on-chain properties listed in `FungibleTokenMetadata`.
```

Comment:
These drawbacks don't seem to make sense with the current version of the spec. 

`icon` has been added, so we don't need a note about `icon_url`. Maybe we still need to mention `icon_hash`, especially if we are allowing `icon` to be an HTTP endpoint instead of a data URL. And if we're allowing an HTTP endpoint, then we should definitely mention the security & privacy implications in Drawbacks!

Reply:
Good point, this is from when I wrote all the standards on a Friday, before we added `icon`. I'll remove it, thanks.

Reply:
> And if we're allowing an HTTP endpoint, then we should definitely mention the security & privacy implications in Drawbacks!

yeah, main implication gonna be NEAR Wallet not showing icon for your token if it requires querying untrusted server